### PR TITLE
Skylake v2 Information take 2

### PR DESCRIPTION
**Pull request by ImmaZoni**
This PR is the second attempt at #440 

This pull request updates the following:

1. **Skylake and Legacy CPU Versions Information:** Added details for Skylake, suitable for newer CPUs (post-2014), and Legacy, for older CPUs (pre-2014).
2. **Miscellaneous Fixes:** Aligns latest and pre-release versions, ensuring consistency throughout the documentation.

### CODE REVIEW

File: docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import styles from '@site/src/pages/index.module.css';
 ### Linux Filesystem Recommendations:
 
 :::note Important Notices for Linux Users
-- **SWAP**:  A swap is necessary only if your system has a low amount of RAM.
+- **SWAP**: A swap file is necessary only if your system has a low amount of RAM.
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx:10]: Didn't know that on Linux you have to create a file for that, good idea to add this.

File: docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx [hunk 2]
```diff
@@ -99,7 +99,7 @@ import styles from '@site/src/pages/index.module.css';
 ```
 
 :::note
-Using **run**, setting **--base-path** and specifying **--chain** became mandatory starting with Gemini 3h. 
+Using **run**, setting **--base-path** and specifying **--chain** became mandatory starting with Gemini 3h. The parameter `--validator` was switched to `--farmer` at the same time.
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx:102]: Hm, we had a similar sentence in this note before. Sounds like a we might have a problem of not working of the most up to date master, it's the only reason this message could have disappeared.

File: versioned_docs/version-latest/participate/community_resources/community-maintained-tools.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ keywords:
     - Community built tools
 ---
 
-### Community-maintained list of userful, third-party tools
+### Community-maintained list of useful, third-party tools
```
Comment (EmilFattakhov) [inline versioned_docs/version-latest/participate/community_resources/community-maintained-tools.md:10]: 😅

**Review by EmilFattakhov — APPROVED** [review]
Looks good Justin!

**Review by EmilFattakhov — CHANGES_REQUESTED** [review]
As per slack discussion, some changes are needed to make it compile fine.

File: docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx (hunk 2 above)
Comment (ImmaZoni) [reply docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx:102]: > Hm, we had a similar sentence in this note before. Sounds like a we might have a problem of not working of the most up to date master, it's the only reason this message could have disappeared.

This arrose from me running update-version command to bring pre-release and latest inline. Looks like it was still present on the other version. Fixed now!

**Review by EmilFattakhov — APPROVED** [review]
Let's hope it compiles 😅

**Pull request merged**